man/man5/zfs-module-parameters.5

```diff
@@ -513,7 +528,7 @@ Default value: \fB100\fR.
 \fBzfs_arc_min_prefetch_lifespan\fR (int)
 .ad
 .RS 12n
-Min life of prefetch block
+Minimum time prefetched blocks are locked in the ARC, specified in seconds.
```

Comment:
This value is actually set in ticks/jiffies (which is terrible but that's another issue).  The default value is 0 which defaults to 1 second (1 * HZ).